Misc/ACKS

```diff
@@ -1599,6 +1599,7 @@ Erik Rose
 Mark Roseman
 Josh Rosenberg
 Jim Roskind
+Prince Roshan
```

Comment:
Can you add this entry one line above (h coming before k)

Reply:
Done